slidey_js.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function(){
+
+images = ["http://i.imgur.com/gmLyvbp.jpg", "http://i.imgur.com/tL55PPP.jpg", "http://i.imgur.com/ntuo5zU.jpg", "http://i.imgur.com/hYK4gEX.jpg", "http://i.imgur.com/CH2fQyB.jpg", "http://i.imgur.com/uO8nSk1.jpg"];
+back= $("#background");
+num = 0;
+
+setInterval(function(){
+	back.css({
+		"background-image": "url('"+ images[num] + "')",
+		"background-size": "80% 80%"
```

Comment:
excellent solution! Try moving styles to a stylesheet so you won't have to update your JS if you want to change the styles.